### ISSUE TITLE
Incomplete partition of the cochlea HP:0011373 textual definition and EQ

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/human-phenotype-ontology#7389]
While reviewing MP-HPO logical mappings for decreased cochlear coiling [MP:0003148](http://purl.obolibrary.org/obo/MP_0003148) and Incomplete partition of the cochlea [HP:0011373](http://purl.obolibrary.org/obo/HP_0011373), both EQs were shown to use:

has part some (decreased coiling and (characteristic of some cochlea) and (has modifier some abnormal)) 

I am uncertain as to whether the HPO textual definition matches the EQ. 

Could you please clarify the textual definition for the HPO term so that I can determine if the logical mapping is correct?

In other words, is the formation of the cochlea partition formed by coiling of the cochlea?

Thanks so much, 

Anna

**Comment by pnrobinson** [comment]
I think the EQ is incorrect. @matentzn @LCCarmody do we have a pattern for the incomplete formation of a partition/septum that is normally there and its partial absence is the abnormality?

**Comment by matentzn** [comment]
I assigned @rays22 (@pnrobinson @rays22 is our new EQ expert and can be tagged on all EQ related tickets) to respond and help @LCCarmody to determine a solution.

**Comment by pnrobinson** [comment]
@rays22 can we touch bases about this?

**Comment by matentzn** [comment]
Is this still relevant?

**Comment by LCCarmody** [comment]
I have not worked on it. cursory look, the text definition could be expanded, but I don't think the human version of this exactly matches the mouse phenotype. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635626/#:~:text=6%2D%20Incomplete%20Partition%20of%20the,and%20various%20internal%20architecture%20defects.

**Comment by pnrobinson** [comment]
Coming back to this, I do not think it is a great idea to have the logical definition reflect our presumption about the pathophysiology (coiling). We should create a PATO term for incompletely partitioned and use it for MP and HP. @matentzn @rays22 @LCCarmody

**Comment by LCCarmody** [comment]
There is Incomplete structure (PATO:0005013). I am not sure if that would work.

**Comment by rays22** [comment]
I am fine with using either Incomplete structure (PATO:0005013) or a newly created PATO incompletely partitioned quality in the EQ.
Let me know if I should create a term request (and a PR) in PATO for incompletely partitioned.

**Comment by pnrobinson** [comment]
@LCCarmody @rays22 
I would favor Incomplete structure because developmental defects like this are extremely complex and it is not going to be possible to model them precisely with OWL.

**Comment by anna-anagnostop** [comment]
Yes, but we will need to request a new pattern for ‘incomplete structure’ and discuss where it is approropriate to use it.
ZP and XPO already use [PATO:0005013](http://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0005013) , see otolith incomplete structure, abnormal [ZP:0022484](http://www.ebi.ac.uk/ols4/ontologies/zp/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FZP_0022484) and 
abnormally incomplete structure of forebrain [XPO:0115310](http://www.ebi.ac.uk/ols4/ontologies/xpo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FXPO_0115310)

**Comment by LCCarmody** [comment]
I'll create a ticket in uPHENO and we can discuss this at a meeting for specifics.

**Comment by pnrobinson** [comment]
@LCCarmody @anna-anagnostop @rays22 can we move forward with this?
